.travis.yml

```diff
@@ -6,6 +6,7 @@ python:
     - "2.7"
 
 sudo: false
+dist: trusty
```

Comment:
While I understand the desire to make this change, I think it's out of scope of this Pull Request and a separate PR should be made for it or we should wait until Travis issues are resolved.

Reply:
drive by: this change already exists on develop now, so can be removed here -